src/driver-steelseries.c

```diff
@@ -440,98 +508,84 @@ steelseries_write_led_v2(struct ratbag_led *led)
 {
 	struct ratbag_device *device = led->profile->device;
 	uint8_t buf[STEELSERIES_REPORT_SIZE] = {0};
-	uint16_t duration, min_duration;
 	int ret;
 
-	buf[0] = STEELSERIES_ID_LED;
-	buf[2] = led->index;
+	struct steelseries_cycle cycle;
+	struct steelseries_point point1;
+	struct steelseries_point point2;
+	struct steelseries_point point3;
+	struct steelseries_point point4;
```

Comment:
an array would be better here tbh, makes things more obvious than separate variables.

Reply:
Yes, I should have done that, sorry.